jekyll/_cci2/workflows.adoc

```diff
@@ -188,16 +147,37 @@ workflows:
             - acceptance_test_4
 ----
 
-In this example, as soon as the `build` job finishes successfully, all four acceptance test jobs start. The `deploy` job must wait for all four acceptance test jobs to complete successfully before it starts.
+In this example, as soon as the `build` job finishes successfully, all four acceptance test jobs start. The `deploy` job waits for all four acceptance test jobs to succeed before it starts.
 
 See the link:https://github.com/CircleCI-Public/circleci-demo-workflows/tree/fan-in-fan-out[Sample Fan-in/Fan-out Workflow config] for a full example.
 
 [#holding-a-workflow-for-a-manual-approval]
 == Hold a workflow for a manual approval
 
-Configure a workflow to wait for manual approval before continuing using an `approval` job. Anyone who has push access to the repository can click the *Approve* button in the CircleCI web app to continue the workflow.
+Use an `approval` job to configure a workflow to wait for manual approval before continuing. Anyone who has push access to the repository can approve the job to continue the workflow. Approve either by using the *Approve* button in the CircleCI web app, or via the API.
+
+Some things to keep in mind when using manual approval in a workflow:
+
+* `hold` is a special job type that is configured when listing jobs under the `workflow` key. You do not need to define a `hold` type job in the `jobs` section of your configuration. If you do configure steps for a job that is given the `hold` type in the workflows section, the steps for that job will not be run. A `hold` job is _only_ used to hold the workflow for approval, not to run any work.
```

Comment:
`hold` is not a job type.  `approval` is the job type that can be configured. So I think referring to `hod` as a job type is confusing.

I think `hold` is better described as the UI label/representation of an approval job.